### PR TITLE
Lint maintainers

### PR DESCRIPTION
**Pull request by kytrinyx**
Now that we have a `maintainers.json` file, we're inevitably going to find that we delete fields without deleting the final comma. Or that someone adds the file to a track with a single maintainer, and makes `maintainers` an object rather than an array.

This includes the `maintainers.json` in the linting process.

### CODE REVIEW

File: track/maintainer_config.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package track
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type MaintainerConfig struct {
```
Comment (nywilken) [inline track/maintainer_config.go]: Missing top level comment for exported types.

File: track/maintainer_config.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	ShowOnWebsite bool   `json:"show_on_website"`
+	Alumnus       bool   `json:"alumnus"`
+	Name          string `json:"name"`
+	Bio           string `json:"bio"`
+	LinkText      string `json:"link_text"`
+	LinkURL       string `json:"link_url"`
+	AvatarURL     string `json:"avatar_url"`
+}
+
+func NewMaintainerConfig(path string) (MaintainerConfig, error) {
```
Comment (nywilken) [inline track/maintainer_config.go]: Same linting issue as exported types; missing comment.

**Review by nywilken — CHANGES_REQUESTED** [review]
This looks good some minor liniting issues regards the exported types/func.

File: track/maintainer_config.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	mc := MaintainerConfig{}
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```
Comment (nywilken) [inline track/maintainer_config.go]: Very nice!

File: track/maintainer_config_test.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package track
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBrokenMaintainerConfig(t *testing.T) {
```
Comment (nywilken) [inline track/maintainer_config_test.go:9]: I like this test pattern. Short, simple, and clearly showcases how linting for the maintainers file works.

File: track/maintainer_config.go (hunk 1 above)
Comment (kytrinyx) [reply track/maintainer_config.go]: Ah, I totally forgot to run the linter. My bad!

**Comment by kytrinyx** [comment]
I've added the linting. Merging this in. Thanks @nywilken

**Pull request merged**